runtime/commonMain/src/kotlinx/serialization/SealedSerializer.kt

```diff
@@ -78,7 +79,7 @@ public class SealedClassSerializer<T : Any>(
     private val inverseMap: Map<String, KSerializer<out T>>
 
     init {
-        require(subclasses.size == subclassSerializers.size) { "Arrays of classes and serializers must have the same length" }
+        require(subclasses.size == subclassSerializers.size) { "Arrays of classes and serializers must have the same length, got arrays: $subclasses, $subclassSerializers" }
```

Comment:
1) The line is too long
2) `toString` on arrays does not help here

Reply:
https://youtrack.jetbrains.com/issue/KT-34776